### PR TITLE
Mark embedded doc

### PR DESCRIPTION
**Pull request by xieWright**
Updated remaining Doxygen comments in Shaders, Lighting, and some in Window.

### CODE REVIEW

**Review by criticalglitch — APPROVED** [review]

**Review by bakursait2 — CHANGES_REQUESTED** [review]
- All editing looks perfect. However, I think we are missing:
  - [Issue#42](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/42) in `model.handle_secondary_action()`
  - [Issue#57](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/57) in `model.collide()`
  - [Issue#82](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/82) in `window.handle_flight()`
  - [Issue#22](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/22) in `window.on_mouse_press()`
  - [Issue#22](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/22) in `window.on_mouse_motion()`
  - change Issue number from [#38](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/38) to [Issue#37](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/37) in `player.speed_up()`
- We need a description for class `Block`. I thought I added it in my branch but apparently I forgot. Sorry for that.

Thanks for your effort.

**Comment by xieWright** [comment]
Finished writing the comments for the missing functions in model. However, since world hasn't been merged yet, I can't do those yet.

**Review by w1158mxs — CHANGES_REQUESTED** [review]
Didn't pass test, Need to be fixed

**Comment by xieWright** [comment]
Edited it back so that the tests would pass. However, I think handle_primary_action and handle_secondary_action should get a name change in the future to reflect that it's destroying and building blocks.

**Comment by w1158mxs** [comment]
> * All editing looks perfect. However, I think we are missing:
>   
>   * [Issue#42](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/42) in `model.handle_secondary_action()`
>   * [Issue#57](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/57) in `model.collide()`
>   * [Issue#82](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/82) in `world.handle_flight()`
>   * [Issue#22](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/22) in `world.on_mouse_press()`
>   * [Issue#22](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/22) in `world.on_mouse_motion()`
>   * change Issue number from [#38](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/38) to [Issue#37](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/37) in `player.speed_up()`
> * We need a description for class `Block`. I thought I added it in my branch but apparently, I forgot. Sorry for that.
> 
> Thanks for your effort.

Never mind, I did them all!

File: tempus_fugit_minecraft/window.py [hunk 1]
```diff
@@ -414,7 +414,8 @@ def draw_reticle(self) -> None:
 
     def update_day_night(self, dt: float) -> float:
         """!
-        @brief Updates the environments lights.
+        @brief Updates the environments lights. When time elapses, the lighting will change.
+            from the ingame time between 0-11 light decreases while from 12-23 light increases
```
Comment (w1158mxs) [inline tempus_fugit_minecraft/window.py:418]: in game

**Review by w1158mxs — APPROVED** [review]

File: tempus_fugit_minecraft/sound.py [hunk 2]
```diff
@@ -1,37 +1,35 @@
 from pyglet import media
 
 
+#issue 17
```
Comment (Lurk390) [inline tempus_fugit_minecraft/sound.py:4]: This is redundant with the docstring in the init

File: tempus_fugit_minecraft/sound.py [hunk 3]
```diff
@@ -1,37 +1,35 @@
...
-    @return rock_hit_sound An instance of Sound class.
-    """
     def __init__(self, file_path=None):
-        """!
-        @brief Initializes the Sound class.
-        @details If a file path is named, uses that to load the sound file
-        @param file_path string The file path for the sound file
-        @see [Issue#17](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/17)
-        """
+        '''!Initializes the class Sound. If a file path is named, uses that to load the sound file
```
Comment (Lurk390) [inline tempus_fugit_minecraft/sound.py:7]: Keep `"""!` on a newline for readability

File: tempus_fugit_minecraft/sound.py [hunk 4]
```diff
@@ -1,37 +1,35 @@
...
         if file_path is not None:
             self.sound_file = media.load(file_path)
 
     def load_sound(self, file_path):
-        """!
-        @brief Loads the sound into the media player.
-        @param file_path string The file path for the sound file
-        @see [Issue#17](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/17)
-        """
+        '''!Loads the sound into the media player
```
Comment (Lurk390) [inline tempus_fugit_minecraft/sound.py:22]: Keep `"""!` on a newline for readability

File: tempus_fugit_minecraft/sound.py [hunk 5]
```diff
@@ -1,37 +1,35 @@
...
+            @see [issue#17] (https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/17)
+        '''
         self.sound_file = media.load(file_path)
 
     def play_sound(self):
-        """!
-        @brief plays the sound contained in the class.
-        @see [Issue#17](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/17)
-        """
+        '''!Plays the sound file contained in the class
```
Comment (Lurk390) [inline tempus_fugit_minecraft/sound.py:29]: Keep `"""!` on a newline for readability

File: tempus_fugit_minecraft/sound.py [hunk 6]
```diff
@@ -42,10 +40,14 @@ def play_sound(self):
...
 
 class BackgroundSound(Sound):
-    """!
-    @brief Subclass of sound for sounds that will be repeated
-    @return wind_blowing An instance of BackgroundSound class.
-    """
+    '''Subclass of sound for sounds that will loop'''
     def __init__(self, file_path=None):
         super().__init__(file_path)
+        '''!Initializes the class BackgroundSound. This is to separate sounds that will loop and have different
```
Comment (Lurk390) [inline tempus_fugit_minecraft/sound.py:46]: Keep `"""!` on a newline for readability

**Review by Lurk390 — CHANGES_REQUESTED** [review]
Looks good overall, just some formatting changes

**Review by bakursait2 — CHANGES_REQUESTED** [review]
> Finished writing the comments for the missing functions in model. However, since world hasn't been merged yet, I can't do those yet.
Sorry, I made a mistake when I mentioned a change to `world.on_mouse_press()` because it is not there yet. I meant to make those changes on `window.on_mouse_press()`.

Also the above amend request has not been resolved. Here I am mention it:
- [Issue#57](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/57) in `model.collide()`
- [Issue#82](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/82) in `window.handle_flight()`
- [Issue#22](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/22) in `window.on_mouse_press()`
- [Issue#22](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/22) in `window.on_mouse_motion()`
- change Issue number from [#38](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/38) to [Issue#37] in `player.speed_up()`

I know this is not in your PR plan but I believe since you are adding comments and links on the same files that I want these updates to be added, then I believe that would be fine.

Thank you.

**Review by bakursait2 — APPROVED** [review]

**Review by Lurk390 — CHANGES_REQUESTED** [review]
Everything looks good to me, however, the tests are still saying they failed on my end.

**Comment by xieWright** [comment]
Yeah, found a syntax error, fixed it.

**Review by Lurk390 — APPROVED** [review]

**Pull request merged**